ui/src/app/views/recent-work/component.spec.ts

```diff
@@ -102,7 +102,7 @@ describe('RecentWorkComponent', () => {
     expect(cardsOnPage.length).toEqual(4);
     // should match LAST 4, and NOT include the "oldest"
     expect(cardNames).toEqual(
-        ['mockFile5.ipynb', 'mockFile4.ipynb', 'mockFile3.ipynb', 'mockFile2.ipynb']);
+        ['mockFile5', 'mockFile4', 'mockFile3', 'mockFile2']);
```

Comment:
I'm not sure whether this was intentional, but if so could you please split this change out? THis feels like an entirely separable 3rd PR in one.

Reply:
When doing this ticket I noticed that the mocks and what was on the site differed, I checked with Karthik and this is the way he wants it. Since I was doing a lot of tweaking/fixing of how we were dealing with notebook names these tests were failing. So I fixed them.

Reply:
I guessed as much. I don't think it has any real relationship to this PR other than you happened to be working on these things at the same time, so it would be cleaner to just send a separate PR for it.